spec.bs

```diff
@@ -18,3 +18,122 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="advertisements">Advertisements</h2>
+
+An {{AuctionAd}} contains the following:
+
+* A <dfn for=AuctionAd dfn-type="dict-member">renderUrl</dfn> .
+* An optional <dfn for=AuctionAd dfn-type="dict-member">metadata</dfn> which is text containing opaque JSON encoded data.
+
+<xmp class="idl">
+dictionary AuctionAd {
+  required USVString renderUrl;
+  any metadata;
+};
+</xmp>
+
+
+<h2 id="advertisement-interest-groups">Advertisement Interest Groups</h2>
+
+An <dfn export>advertisement interest group</dfn>
+is a [=dictionary=] with the following:
+ 
+* An <dfn export for="AuctionAdInterestGroup">expiration</dfn> point in time.
+
+* An <dfn export for="AuctionAdInterestGroup">owner</dfn> [=origin=].
+
+* A <dfn export for="AuctionAdInterestGroup">name</dfn>, a [=string=].
+ 
+* A <dfn export for="AuctionAdInterestGroup">priority</dfn>, a {{double}}, initially 0.
+ 
+* An <dfn export for="AuctionAdInterestGroup">enableBiddingSignalsPrioritization</dfn>, a [=boolean=], initially false.
+ 
+* An optional <dfn export for="AuctionAdInterestGroup">priorityVector</dfn>, A [=map=] from [=string=] to {{double}}.

```

Comment:
Yes.

Reply:
Well, each map entry is from a singular string to a singular double.  It would be ambiguous if the plural form implied an array, but in that case, it should be list of double, or would that be a list of doubles?   I'd rather be consistent by always using the singular form, but I'll make them plural for now.